.github/workflows/release_mac.yml

```diff
@@ -52,14 +53,19 @@ jobs:
       run: |
         # Install Protobuf from source
         export NUM_CORES=`sysctl -n hw.logicalcpu`
+        if [ '${{ matrix.target-architecture }}' == 'x86_64' ]; then
+          export CMAKE_OSX_ARCHITECTURES=x86_64
+        elif [ '${{ matrix.target-architecture }}' == 'universal2' ]; then
+          export CMAKE_OSX_ARCHITECTURES='arm64;x86_64'
+        fi
         source workflow_scripts/protobuf/build_protobuf_unix.sh $NUM_CORES $(pwd)/protobuf/protobuf_install
         export CMAKE_ARGS="-DONNX_USE_LITE_PROTO=ON"
         # Currently GitHub Action agent is using macos-11, use -p to force change the final MACOSX_DEPLOYMENT_TARGET
         # Change -p if MACOSX_DEPLOYMENT_TARGET is different
         if [ '${{ github.event_name }}' == 'schedule' ]; then
-          python setup.py bdist_wheel -p macosx_10_12_x86_64 --weekly_build
+          python setup.py bdist_wheel -p macosx_10_12_${{ matrix.target-architecture }} --weekly_build
         else
-          python setup.py bdist_wheel -p macosx_10_12_x86_64
+          python setup.py bdist_wheel -p macosx_10_12_${{ matrix.target-architecture }}
```

Comment:
nit: not sure whether we want to use `macosx_10_12_` for universal2 as well. Probably we can assume only 10.15+ users will try universal2 wheel.

Reply:
Thanks for your review! It is what I ignored.

What's the reason for choosing 10.15? The minimum macos version that can run on apple silicon chips seems 11. Maybe we can use `macosx_11_0_`.

Reply:
Good point. `macosx_11_0_` looks better. Thanks!

Reply:
Updated. Waiting for CI :)

Reply:
@jcwchen I found that python3.7 is incompatible with `macosx_11_0_`.

```bash
$ python3.7 -m pip install onnx-1.12.0-cp37-cp37m-macosx_11_0_x86_64.whl
ERROR: onnx-1.12.0-cp37-cp37m-macosx_11_0_x86_64.whl is not a supported wheel on this platform.
$ python3.7 -m pip install onnx-1.12.0-cp37-cp37m-macosx_10_15_x86_64.whl
Processing ...
$ python3.8 -m pip install onnx-1.12.0-cp38-cp38-macosx_11_0_x86_64.whl
Processing ...
```

So I prefer keeping the versions of x86_64 and universal2 wheels the same. It is also what [numpy](https://pypi.org/project/numpy/1.21.0/#files) does:

<img width="564" alt="image" src="https://user-images.githubusercontent.com/11607199/201507644-78e890ab-49d8-49b5-9829-f5d0c99ef84d.png">